core/pom.xml

```diff
@@ -19,6 +19,46 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-webflux</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.5</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
How jUnit version has been chosen?

Reply:
I think 4 version are more popular, and used in our project